### DISCUSSION TITLE
no me reconoce mingw

### DISCUSSION THREAD

**Discussion by Juampi44** [INGCOM-UNRN-P1/cursada-2023#8] [Preguntas y respuestas] [answered]
Cuando escribo el comando `$> gcc --version` o `gcc --version` en la consola me da como respuesta "no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.". Ya instale mingw pero no se que mas tengo que hacer

**Comment by martinvilu** [comment]
Cuál estás usando, ¿el que fue dado en clases? ¿o la guía que está en el wiki?

**Reply by Juampi44** [reply]
Instale uno que vi en un video porque el del wiki cuando lo descargue windows me decia que el archivo estaba roto y el pendrive no supe como instalarlo

**Reply by martinvilu** [reply]
Bueno, que descargaste y como lo instalaste?

**Reply by martinvilu** [reply]
Mirá la parte del wiki que habla sobre el "`path`" en particular

**Reply by Juampi44** [reply]
https://www.youtube.com/watch?v=GEZLjRSY_m8 de ese video lo instale, ahora miro lo del path

**Reply by martinvilu** [reply, marked as answer]
Para lo del `path` tenés que tener ubicado donde está descompactado/instalado el `mingw`. Que debiera de ser una 'dirección' simple, como `c:\mingw`. (aunque __técnicamente__ podés usar cualquier dirección, esto te *va* a ahorrar dolores de cabeza mas adelante)

**Reply by Juampi44** [reply]
okay, gracias, ahora me fijo
